### DISCUSSION TITLE
Different models yield the same exact output?

### DISCUSSION THREAD

**Discussion by JonAnderRuizColmenares** [shap/shap#3667] [Q&A]
Hello,

Maybe I am misunderstanding or this is completely intended for SHAP values, but I train three simple gradient regressors (XGBoost, CatBoost, LightGBM) with my data and then I apply the Explainer and obtain the shap values.

The three models predict identically, ( I can visualize that with decision plots for example), When I study the general shap structure with beeswarm plots or I get the absolute mean value for importance plots, the values I get are the exact same.

Is this just because my data is the same for all three? Is there any logic to what is happening. Any help would be greatly appreciated, thanks!
